src/core/ExceptionFormatter.js

```diff
@@ -51,6 +55,32 @@ getJasmineRequireObj().ExceptionFormatter = function(j$) {
  
       return result;
     }
+
+    function formatProperties(error) {
+      if (!(error instanceof Object)) {
+        return;
+      }
+
+      var ignored = ['name', 'message', 'stack', 'fileName', 'sourceURL', 'line', 'lineNumber', 'stack'];
+      var result = {};
+      var empty = true;
+
+      for (var prop in error) {
+        if (ignored.includes(prop)) {
+          continue;
+        }
+        result[prop] = error[prop];
+        empty = false;
+      }
+
+      if (!empty) {
+        try {
+          return 'error properties: ' + JSON.stringify(result, null, 2) + '\n';
```

Comment:
I think we probably just want to use Jasmine's own pretty printer here, which will still print something out even with a cycle in the object.